tools/pull-mdi-icons.js

```
@@ -88,36 +94,36 @@
 		const svgZipPath = `${workspace}/${flavour}-svg.zip`;
 		const svgExtractedZipPath = `${workspace}/${flavour}-svg`;
-		fs.mkdirSync(svgExtractedZipPath);
-		await upstream.downloadSvgZip(svgZipPath, flavour, version);
+		mkdirSync(svgExtractedZipPath);
+		await downloadSvgZip(svgZipPath, flavour, version);
 		log(`Downloaded SVG ZIP ✔️`);
-		await upstream.extractZip(
+		await extractZip(
 			svgZipPath,
 			svgExtractedZipPath,
 			`^MaterialDesign(Light)?-SVG-${escapedVersion}\/svg\/.+$`
```

ESLint

[error] Unnecessary escape character: \/.

(no-useless-escape)